storm/Core.hpp

```diff
@@ -13,6 +13,11 @@
 #endif
 #endif
 
+#if !defined(WHOA_SYSTEM_WIN)
+typedef void* HANDLE;
```

Comment:
Keeping this around since presumably `StormLib` will want to work with it. Not sure if we're intending to bring in `StormPort.h`, so we can throw this out if it's not an issue.